tasks.py

```diff
@@ -666,8 +666,13 @@ def _replace(pattern):
 @task
 def test(ctx):
     """Run unit tests."""
-    run("go test -short ./...")
-    run("go test -short -race ./...")
+    envtest_asset_dir = os.getcwd() + "/testbin"
+    run('sudo rm -rf "%s"' % envtest_asset_dir)
+    os.mkdir(envtest_asset_dir)
```

Comment:
what is this for?

Reply:
this is for setting up envtest cluster as it is needed for running `TestNodeEvent` test.

Reply:
yes, done.